test/suite/wttest.py

```diff
@@ -510,7 +510,7 @@ def tearDown(self, dueToRetry=False):
                     else:
                         teardown_msg += "; " + str(tmp[1])
 
-        passed = not (self.failed() or teardown_failed)
+        passed = "None" if self.skipped else not (self.failed() or teardown_failed)
```

Comment:
Is this to make the print below more readable? All of the other uses seem to also consider `skipped`.

Reply:
Yeah, that's the intention. Alternatively we could print `passed=False` when `skipped=True` but I thought that was somehwat misleading.

Reply:
Yeah I agree. I like the approach in this PR. Thanks for the explanation!